onnx/setup.sh

```diff
@@ -0,0 +1,55 @@
+# ----------------------------------------------------------------------------
+#
+# Package	: ONNX
+# Version	: 1.4
+# Source repo	: https://github.com/onnx/onnx.git
+# Tested on	: ubuntu 18.04
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Chin Huang <chhuang@us.ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             A non-root user must be created with sudo permissions.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+export NUMCORES=`grep -c ^processor /proc/cpuinfo`
+if [ ! -n "$NUMCORES" ]; then
+  export NUMCORES=`sysctl -n hw.ncpu`
+fi
+echo Using $NUMCORES cores
+
+sudo apt-get update 
+
+sudo apt-get install -y \
+         protobuf-compiler \
+         libprotobuf-dev \
+         cmake
+
+# use python3 venv based on the env variable PYTHON_VERSION
+if [ "${PYTHON_VERSION}" == "python3" ]; then
+    sudo apt-get install -y \
+         python3-pip \
+         python3-venv
+    python3 -m venv venv_py3
+    source venv_py3/bin/activate
+else
+    sudo apt-get install -y \
+         python-pip 
+fi
+
+python -V
+pip -V
+
+# need to clone source when not running the scripts in Jenkins
+if [ -z "${JENKINS_URL}" ]; then
+    apt install -y git
```

Comment:
```suggestion
    sudo apt install -y git
```

Reply:
@tedhtchang Thanks for the review. A new version is pushed with updates.